app/controllers/teachers_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TeachersController < ApplicationController
+  before_action :set_teacher, only: [:show, :edit, :update, :destroy]
+
+  # GET /teachers
+  # GET /teachers.json
+  def index
+    @teachers = Teacher.all
+  end
+
+  # GET /teachers/1
```

Comment:
You want to remove unnecessary comments in production code. The code should speak for itself.